### DISCUSSION TITLE
Schema handling

### DISCUSSION THREAD

**Discussion by arkiaman** [Mongey/terraform-provider-confluentcloud#73] [Ideas]
Hi @Mongey, I'll be happy to contribute to this repo. I'm currently starting on a project from scratch, and I would need to manage topics, ACLs, and schemas as well.
What are your plans about managing schemas ? Within this provider,  apart, not at all ?

**Comment by Mongey** [comment]
For managing Topics and ACLs, there's [terraform-provider-kafka](https://github.com/Mongey/terraform-provider-kafka)

**Reply by arkiaman** [reply]
I know, but my question was more about schemas...
